ofl/robotoflex/article/ARTICLE.en_us.html

```diff
@@ -1,5 +1,5 @@
-<a href="https://youtu.be/f3IQSmKFokU"><img src="hero.png" /></a>
-<p><em><a href="https://youtu.be/f3IQSmKFokU">Watch the introduction video on YouTube</a></em</p>
+<a href="https://youtu.be/f3IQSmKFokU"><img src="hero.jpg" /></a>
```

Comment:
The jpg to what this points at now needs to be included

Reply:
Ah yes, I didn't finish the image processing task I have open. Moving to draft until this is ready :)